pkg/webui/console/containers/gateway-overview-header/gateway-overview-header.styl

```
@@ -0,0 +1,50 @@
+// Copyright © 2024 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.root
+  position: sticky
+  top: 0
+  z-index: calc($zi.dropdown - 1)
+  display: flex
+  justify-content: space-between
+  gap: $cs.xl
+  align-items: center
+  padding: $cs.m
+  background-color: $c['tts-primary-050']
```

```suggestion
  background-color: var(--bg-neutral-min)
```
Let's use this actually. I've updated the design as well.